app/assets/stylesheets/organisations.scss

```diff
@@ -0,0 +1,7 @@
+// Place all the styles related to the Organisations controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+table, thead, th, td, tr {
```

Comment:
This affects all tables. Shouldn't it be in the main css?

Reply:
Sounds good. Will add it to the main css. Was just trying out, didn't know where it was supposed to go :stuck_out_tongue: